wiki/Storyboard_Scripting/Cheat_Sheet/en.md

```diff
@@ -41,24 +41,25 @@ Z-order (back to front) is determined by the order the files appear in the .osu
 
 **layer**:
 
--   Background
--   Fail
--   Pass
--   Foreground
+-   0 - Background
+-   1 - Fail
+-   2 - Pass
+-   3 - Foreground
 
 *Note: Background and Foreground are always visible.*
 
 **origin**:
 
--   TopLeft
--   TopCentre
--   TopRight
--   CentreLeft
--   Centre
--   CentreRight
--   BottomLeft
--   BottomCentre
--   BottomRight
+-   0 - TopLeft
+-   1 - Centre
+-   2 - CentreLeft
+-   3 - TopRight
+-   4 - BottomCentre
+-   5 - TopCentre
+-   6 - Custom (same effect as TopLeft but shouldn't be used)
```

Comment:
comma before but
shouldn't -> should not

Reply:
This was fixed over a month ago... maybe you should stop blocking? >.>